WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInput.swift

```diff
@@ -2,13 +2,87 @@ import SwiftUI
 import Yosemite
 
 struct ShippingLabelCustomsFormInput: View {
+    private let isCollapsible: Bool
+    private let packageNumber: Int
+    private let safeAreaInsets: EdgeInsets
+
+    @ObservedObject private var viewModel: ShippingLabelCustomsFormInputViewModel
+    @State private var showingContentTypes = false
+
+    init(isCollapsible: Bool, packageNumber: Int, safeAreaInsets: EdgeInsets, viewModel: ShippingLabelCustomsFormInputViewModel) {
+        self.isCollapsible = isCollapsible
+        self.packageNumber = packageNumber
+        self.safeAreaInsets = safeAreaInsets
+        self.viewModel = viewModel
+    }
+
     var body: some View {
-        Text("Hello, World!")
+        CollapsibleView(isCollapsible: isCollapsible, safeAreaInsets: safeAreaInsets, label: {
+            headerView
+        }, content: {
+            VStack {
```

Comment:
I think we need to set the `spacing` to 0 here, to avoid extra space (especially between the rows and dividers):

```suggestion
            VStack(spacing: 0) {
```

Reply:
Great catch! I found this issue too and have updated it in #4772.